tools/loop-back-lvm/entrypoint.sh

```diff
@@ -10,5 +10,5 @@ rm /host/create_lvm.sh
 echo "done" >/ready
 echo "ready"
 while true; do
-  sleep 60
+  sleep 1
```

Comment:
Any reason to keep this running not that it's a pod?  I see no issue with keeping a completed pod around

Reply:
Not quite sure why, but if I do an exit 0 to make a clean exit, it ends up in a crash loop...

Reply:
Maybe try removing the readiness probe?

Reply:
And maybe restart policy of Never?

Reply:
Yeah that appears to work fine. Pushed an update.